### PR TITLE
MC-1744 Documentation for Sets and Lists pages

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

File: docs/modules/data-structures/pages/list.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+= Lists
+[[managing-lists]]
+
+You can see a list of all the lists in your cluster
+by clicking on the **Lists** menu item on the left panel. A new
+page is opened on the right, as shown below.
+
+image:ROOT:ListGridView.png[List Grid View]
```
Comment (promanenko) [inline docs/modules/data-structures/pages/list.adoc:8]: From the image it's not clear that `Lists` are in `Storage` block, can you update the screenshot with collapsed `Cluster` block? Also it makes sense to add multiple lists in the example screen.
The same applies to `Sets`

File: docs/modules/data-structures/pages/list.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+You can sort the table by clicking on the column headers.
```
Comment (promanenko) [inline docs/modules/data-structures/pages/list.adoc:10]: please add info about the list sorting/filtering.
Also it makes sense to add general information about List data structure, similar to what we have for MultiMap.
The same applies to Set.

File: docs/modules/data-structures/pages/list.adoc (hunk 1 above)
Comment (Sodik) [reply docs/modules/data-structures/pages/list.adoc:8]: done

File: docs/modules/data-structures/pages/list.adoc (hunk 2 above)
Comment (Sodik) [reply docs/modules/data-structures/pages/list.adoc]: done

**Review by promanenko — APPROVED** [review]

**Pull request merged**